### PR TITLE
17729 - AUTH-WEB - Navigation after account creation with BC Services Card

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/17729

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2713-0147afhn.web.app

File: auth-web/.env.example [hunk 1]
```diff
@@ -18,6 +18,7 @@ VUE_APP_XLP_FORMS_URL="https://www2.gov.bc.ca/assets/gov/employment-business-and
 
 #vaults web-url
 VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
+VUE_APP_REGISTRY_URL="https://dev.bcregistry.gov.bc.ca"
```
Comment (seeker25) [inline auth-web/.env.example:21]: Isn't this the same as the line above?

File: auth-web/devops/vaults.env [hunk 2]
```diff
@@ -13,6 +13,7 @@ VUE_APP_ONE_STOP_URL="op://relationship/$APP_ENV/auth-web/ONE_STOP_URL"
 
 #vaults web-url
 VUE_APP_REGISTRY_HOME_URL="op://web-url/$APP_ENV/registry/REGISTRY_HOME_URL"
+VUE_APP_REGISTRY_URL="op://web-url/$APP_ENV/registry/REGISTRY_URL"
```
Comment (seeker25) [inline auth-web/devops/vaults.env:16]: Are lines 15 and 16 the same URL?

File: auth-web/.env.example (hunk 1 above)
Comment (seeker25) [reply auth-web/.env.example:21]: from PROD:

VUE_APP_REGISTRY_HOME_URL="https://www.bcregistry.gov.bc.ca/"

File: auth-web/devops/vaults.env (hunk 2 above)
Comment (Jxio) [reply auth-web/devops/vaults.env:16]: no, registries home url direct to dev.bcros.ca, registry_url direct to bc registries dashboard

File: auth-web/.env.example (hunk 1 above)
Comment (Jxio) [reply auth-web/.env.example:21]: thanks for point it out, it's same, my local .env HOME_URL direct to https://dev.bcros.ca/, which is wrong, I'll update it.

**Comment by seeker25** [comment]
@Jxio you'll need to resync

File: auth-web/package-lock.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "auth-web",
-  "version": "2.4.66",
+  "version": "2.4.67",
```
Comment (seeker25) [inline auth-web/package-lock.json:3]: you'll have to resync again, 2.5.0 was just merged

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2713-0147afhn.web.app

**Comment by seeker25** [comment]
```
  /**
   *  The url tied to the breadcrumb back button.
   *  Cross application requires href.
   *  Returns to previous route when navigating locally.
   */
  private get backUrl (): string {
    const routeIndex = this.breadcrumbs.findIndex(item => item.to?.name === this.$route?.name)
    return this.breadcrumbs[routeIndex - 1]?.href || this.breadcrumbs[routeIndex - 1]?.to?.name
  }

```

**Pull request merged**